code/scripting/api/libs/graphics.cpp

```diff
@@ -250,6 +250,14 @@ ADE_VIRTVAR(CurrentRenderTarget, l_Graphics, "texture", "Current rendering targe
 	}
 }
 
+ADE_FUNC(clear, l_Graphics, nullptr, "Calls gr_clear().  This is sometimes called between setClip and resetClip", nullptr, nullptr)
```

Comment:
That is not a very useful help string. Better would be something like "Fills the entire screen with the currently active color. Useful if you want to have a fresh start for drawing things."-

Reply:
The documentation I added was the extent of my knowledge. :)  I'll add your suggestion.